driver.go

```diff
@@ -43,4 +43,9 @@ type Driver interface {
 	//
 	// Since: 2.5
 	SetDisableScreenBlanking(bool)
+
+	// Clipboard returns the system clipboard.
+	//
+	// Since: 2.6
+	Clipboard() Clipboard
```

Comment:
This PR exposes the clipboard as a new public API on both the App and the Driver. I think we should just choose one location for the new API - I think the app makes more sense but open to suggestions

Reply:
Good point. I agree that the app makes more sense.